primitives/block-builder/runtime-api/src/lib.rs

```diff
@@ -18,16 +18,19 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use sr_primitives::{traits::Block as BlockT, ApplyResult};
+use sr_primitives::{traits::Block as BlockT, InclusionOutcome};
 
 use inherents::{InherentData, CheckInherentsResult};
 
 sr_api::decl_runtime_apis! {
 	/// The `BlockBuilder` api trait that provides the required functionality for building a block.
 	#[api_version(3)]
 	pub trait BlockBuilder {
-		/// Apply the given extrinsics.
-		fn apply_extrinsic(extrinsic: <Block as BlockT>::Extrinsic) -> ApplyResult;
+		/// Apply the given extrinsic.
+		///
+		/// Returns an inclusion outcome which specifies if this extrinsic should be included in
```

Comment:
Hmm, this "should be". Isn't it more like "could be"? 
In some sense you are right, we still could discard the prospective, but it is more like that the runtime should decide and not the node if a transaction should be included.

Reply:
ehm, doesn't feel difference between these, so will stick with `is`.